src/common/SPELLS/index.ts

```diff
@@ -35,31 +38,29 @@ import SHAMAN from './shaman';
 import Spell, { Enchant } from './Spell';
 import WARLOCK from './warlock';
 import WARRIOR from './warrior';
-import Expansion from 'game/Expansion';
-import { maybeGetSpell as maybeGetClassicSpell } from 'common/SPELLS/classic';
 
-const ABILITIES = {
-  ...OTHERS,
-  ...ENCOUNTER,
-  ...RACIALS,
-  ...DEATH_KNIGHT,
-  ...DEMON_HUNTER,
-  ...DRUID,
-  ...EVOKER,
-  ...HUNTER,
-  ...MAGE,
-  ...ARCANE_MAGE,
-  ...FIRE_MAGE,
-  ...FROST_MAGE,
-  ...MONK,
-  ...PALADIN,
-  ...PRIEST,
-  ...ROGUE,
-  ...SHAMAN,
-  ...WARLOCK,
-  ...WARRIOR,
-  ...DRAGONFLIGHT,
-} as const;
+const ABILITIES = safeMerge(
```

Comment:
i assume this change is what pointed out the issue with evoker vs. dh glide?

Reply:
Yep!